chore: raname endpoint requset to endpoint call, fix typos (#1361)
vaadin/flow-and-components-documentation: documentation/polymer-templates/tutorial-template-parent-layout.asciidoc

---
title: PolymerTemplate, Using Parent Layout
order: 18
layout: page
---

= PolymerTemplate, Using Parent Layout

> *Usage of polymer based templates is deprecated since Vaadin 18 and we recommend using Lit based templates instead.*

Reviewer: - “Polymer” should be capitalised
- Better break onto two separate sentences
- I guess LitElement should be mentioned with a complete name, otherwise it is confusing, as there are also `lit-html` templates

```suggestion
*Usage of Polymer based templates is deprecated since Vaadin 18. It is recommended to use LitElement based templates instead.*
```
